technology_and_tooling/version_control/slides/index.md

```diff
@@ -0,0 +1,176 @@
+---
```

Comment:
This is great, thanks @steve-crouch. I'm a bit unsure about including the html as well. For all the rest of the material we only vc the markdown and the html is generated on the fly using gutenberg. Should we do the same here?

Reply:
I think we should in the long term, but since this functionality isn't in gutenberg atm lets merge it in